projects/swimlane/ngx-charts/src/lib/models/chart-data.model.ts

```diff
@@ -56,18 +58,18 @@ export interface TreeMapDataItem {
   extra?: any;
 }
 
-export interface TreeMapData extends Array<TreeMapDataItem> {}
+export interface TreeMapData extends Array<TreeMapDataItem> { }
 
 export interface BoxChartDataItem extends DataItem {
-  value: number | Date;
+  value: number;
```

Comment:
@KingDarBoja While fixing the conflicts with 6d591944a7eaa2942b4b4c39c9d3526d28ceb1b9, I had to change the type of `value`. I'm not entirely sure of the side-effects this may have.

Reply:
Shouldn't matter as it is only the value that is being displayed in the chart. I think the `number | Date` was due to the original behaviour of the other charts, hence I had some logic to format date objects into strings in the view.